pom.xml

```diff
@@ -138,6 +138,7 @@
         <io.confluent.ksql.version>7.0.0-0</io.confluent.ksql.version>
         <io.confluent.schema-registry.version>${confluent.version.range}</io.confluent.schema-registry.version>
         <mockito.version>3.8.0</mockito.version>
+        <netty-tcnative-version>2.0.29.Final</netty-tcnative-version>
```

Comment:
https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static/2.0.39.Final?

Reply:
Oops, meant to update that.  Changed.